gradio/components/image_editor.py

```diff
@@ -132,6 +132,20 @@ def __post_init__(self):
             self.layers = ["Layer 1"]
 
 
+@document()
```

Comment:
I think we should have the same API across image, video and image editor components for consistency

Reply:
Sounds good, i'll update everywhere.

Reply:
updated!